runtime-extension-model/src/main/java/org/mule/runtime/config/internal/validation/OperationParameterDefaultValueDoesntSupportExpressions.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.internal.validation;
+
+import static org.mule.runtime.api.component.ComponentIdentifier.builder;
+import static org.mule.runtime.api.meta.model.parameter.ParameterGroupModel.DEFAULT_GROUP_NAME;
+import static org.mule.runtime.ast.api.util.ComponentAstPredicatesFactory.currentElemement;
+import static org.mule.runtime.ast.api.validation.Validation.Level.ERROR;
+import static org.mule.runtime.ast.api.validation.ValidationResultItem.create;
+
+import static java.lang.String.format;
+
+import org.mule.runtime.api.component.ComponentIdentifier;
+import org.mule.runtime.ast.api.ArtifactAst;
+import org.mule.runtime.ast.api.ComponentAst;
+import org.mule.runtime.ast.api.ComponentParameterAst;
+import org.mule.runtime.ast.api.validation.Validation;
+import org.mule.runtime.ast.api.validation.ValidationResultItem;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+/**
+ * Expressions are forbidden for optional parameter default values.
+ *
+ * @since 4.5
+ */
+public class OperationParameterDefaultValueDoesntSupportExpressions implements Validation {
+
+  private static final String DEFAULT_EXPRESSION_PREFIX = "#[";
+  private static final String DEFAULT_EXPRESSION_SUFFIX = "]";
+
+  private static final String OPERATION = "operation";
+  private static final String OPTIONAL_PARAMETER = "optional-parameter";
```

Comment:
Shouldn't we take this constants from a common place?
I know it's very unlikely we will ever change them because we would be breaking backwards, but still, it would make it easier to track its usages if we take them from a common place.
Maybe the `MuleOperationExtensionModelDeclarer` could expose them. Any thoughts?

Reply:
Ok, but I would like it more to have it at `OperationDslNamespaceInfoProvider`. Do you agree?

Reply:
Not entirely sure if that's the right place, because it is OperationDsl**Namespace**InfoProvider. I think those providers are for XML-specific stuff.

I still think that these constants belong to the declaration of the ExtensionModel. They could be then consumed from the parsers, validators, etc.

We could involve someone from the SDK team for their opinion.

Reply:
Added to the declarer. Also made a little refactor there to expose some other constants